lib/db_utils.go

```diff
@@ -6736,6 +6903,67 @@ type TransactionMetadata struct {
 	NewMessageTxindexMetadata            *NewMessageTxindexMetadata            `json:",omitempty"`
 }
 
+func (txnMeta *TransactionMetadata) GetEncoderForTxType(txnType TxnType) DeSoEncoder {
```

Comment:
Looks like this is unused?

Reply:
This is used in the consumer code (`txIndexMetadata.GetEncoderForTxType(transaction.TxnMeta.GetTxnType())`), but if you'd prefer to keep it out of core we could move it to a helper in the consumer library.